mj_envs/mujoco_env.py

```diff
@@ -81,7 +81,15 @@ def viewer_setup(self):
         """
         pass
 
+    def get_proprioception(self, use_tactile):
```

Comment:
These are environment specific utilities that you can just add to the env definition. Im not sure if there is any benefit of adding this to the mujoco_env base class.

@aravindr93 - what do you think?

Reply:
Yes, they have to be written on a per-environment basis. But since the function just has pass, it would not break existing code. I'm neutral but can see the value of skeleton definition in the master class.

Reply:
fine with either. @Divye02  its your call which ever you think is a better choice

Reply:
I like this structure since currently all the envs have a get_proprioception functionality.